open-company/open-company-web
# Fix drafts label

Bug: right now all the entry you have in drafts are shown with the _(private)_ label beside the section name. That's because it's using the drafts board access that is private.

To test:
- have a team board, a public board and a private board
- add a post to each of them but don't publish it
- go to drafts
- do you see the right labels beside the section names? Good
- now publish those posts
- do you see the right labels besides the published posts? Good
- edit the private one
- do you still see the right label? Good
